dnf/cli/cli.py

```diff
@@ -985,6 +985,12 @@ def _read_conf_file(self, releasever=None):
         # search config file inside the installroot first
         conf._search_inside_installroot('config_file_path')
 
+        # check, that config file set in CLI, exists
+        filename = conf._get_value('config_file_path')
+        if (conf._get_priority('config_file_path') == dnf.conf.PRIO_COMMANDLINE) and \
+                not os.path.isfile(filename):
+            raise dnf.exceptions.ConfigError(_('Config file "%s" does not exist') % filename)
```

Comment:
This is mostly due to the new style in DNF. I would recommend to use:
```_('Config file "{}" does not exist').format(filename)```

Reply:
@j-mracek: Thank you for the review. I have made the corrections.